src/components/TextField/inputCo.js

```diff
@@ -0,0 +1,116 @@
+import React, { useState } from "react"
+import PropTypes from "prop-types"
+import { makeStyles } from "@material-ui/core/styles"
+import TextField from "@material-ui/core/TextField"
+import InputLabel from "@material-ui/core/InputLabel"
+import FormHelperText from "@material-ui/core/FormHelperText"
+
+import theme from "../../theme/index"
+
+const useStyles = makeStyles(() => ({
+  root: {
+    "& > *": {
+      margin: theme.spacing(1),
+      width: "90%",
+    },
+    "& .Mui-disabled": {
+      backgroundColor: theme.palette.grey[100],
+      color: theme.palette.grey[600],
+    },
+    "& .MuiOutlinedInput-input": {
+      fontSize: "1rem",
+    },
+  },
+  label: {
+    fontSize: "0.875rem",
+  },
+  helperText: {
+    fontSize: "0.75rem",
+    marginTop: "0px",
+    marginLeft: "10px",
+    display: "flex",
+    alignItems: "center",
+  },
+}))
+
+const InputCo = ({
+  id,
+  placeholder,
+  label,
+  value,
+  setValue,
+  helperText,
+  hasErrors,
+  component,
+  ...props
+}) => {
+  const classes = useStyles()
+  const [inputPlaceholder, setPlaceholder] = useState(placeholder)
+  const [errors, setError] = useState("")
+
+  const handleFocus = () => {
+    setPlaceholder("")
+    setError("")
+  }
+  const handelBlur = () => {
+    if (!value) {
+      setPlaceholder(placeholder)
+    }
+    if (hasErrors) {
+      setError(helperText)
+    } else {
+      setError("")
+    }
+  }
+
+  return (
+    <div className={classes.root} noValidate autoComplete="off">
+      {label && (
+        <InputLabel htmlFor={id} className={classes.label}>
+          {label}
+        </InputLabel>
+      )}
+      <TextField
+        {...props}
+        placeholder={inputPlaceholder}
+        label={null}
```

Comment:
Una pregunta. ¿Pasar el `label={null}` te funciona correctamente D:? En mi caso me tomó como si el valor de label fuera null y no me apareciera como quería.

Me tocó ponerle undefined

Reply:
Mmm no tuve problemas. Creeo que si llegas a pasarle undefined puede que en momento de buscarlo con alguna función te salga "cant read bla bla of undefined" y puede romperte el código 🤔